util/tr_openstudio.rb

```diff
@@ -0,0 +1,211 @@
+#!/usr/bin/ruby
```

Comment:
It was 'tr', for 'test runner'. It was suggested I make it more specific to the project to avoid naming conflicts, so I appended the project name.